BrowserKit/Sources/ComponentLibrary/Buttons/LinkButtonViewModel.swift

```diff
@@ -10,15 +10,18 @@ public struct LinkButtonViewModel {
     public let title: String
     public let a11yIdentifier: String
     public let fontSize: CGFloat
-    public let textAlignment: NSTextAlignment
+    public let contentInsets: NSDirectionalEdgeInsets
+    public let contentHorizontalAlignment: UIControl.ContentHorizontalAlignment
 
     public init(title: String,
                 a11yIdentifier: String,
                 fontSize: CGFloat = 16,
-                textAlignment: NSTextAlignment = .left) {
+                contentInsets: NSDirectionalEdgeInsets = .zero,
```

Comment:
Could we have a default `contentInsets` that isn't zero? Maybe we can take the one from `OnboardingCard` as the default for now.

Reply:
Hey, have updated the PR with the same, please have a look.